core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -134,79 +127,18 @@ public void scrollTo(int x, int y) {
     }
 
     @Override
-    protected void onAttachedToWindow() {
-        super.onAttachedToWindow();
-        pagerAdapterObserver.reregister();
+    public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+        scrollingPageChangeListener.onPageScrolled(position, positionOffset, positionOffsetPixels);
```

Comment:
The latter, with the difference that if `ScrollingPageChangeListener` implements the interface you don't need to expose it for the purpose of setting a scroll listener, you can pass around the interface (and not set `this` as listener but the scrolling page listener directly, which is the point I was trying to make)